### ISSUE TITLE
[Bug]: Error: [UNPLAYABLE] Video unavailable: The page needs to be reloaded.

### ISSUE THREAD

**Issue by Freedom-Guda** [FreeTubeApp/FreeTube#8792]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

open app
fails

### Expected Behavior

app is 1000% broken by youtube. google is the devil 

### Issue Labels

accessibility issue

### FreeTube Version

new

### Operating System Version

linux

### Installation Method

Flathub

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by mageofchaos** [comment]
Duplicate of #8779
